src/components/JbrowseLinear.tsx

```diff
@@ -35,7 +31,7 @@ export const JbrowseLinear = ({
   selectedFiles = [],
 }: {
   options?: LinearViewModel;
-  selectedFiles: JbrowseFileInput[];
+  selectedFiles?: JbrowseFileInput[];
```

Comment:
Can't we make this a required property since we provide a default? No reason to have to deal with `undefined` value here.